Source/Core/DiscIO/CISOBlob.cpp

```diff
@@ -5,20 +5,20 @@
 #include <algorithm>
 #include <cstdio>
 #include <memory>
+#include <utility>
 
 #include "Common/CommonTypes.h"
 #include "Common/FileUtil.h"
 #include "DiscIO/CISOBlob.h"
 
 namespace DiscIO
 {
-static const char CISO_MAGIC[] = "CISO";
-
-CISOFileReader::CISOFileReader(std::FILE* file) : m_file(file)
+CISOFileReader::CISOFileReader(File::IOFile&& file) : m_file(std::move(file))
```

Comment:
I forgot that if the `file` argument is a copy, our `std::move` will just move the copy, not the original object that the calling code passes. I'll make the change.